MSAL/src/public/MSALDeviceInformation.h

```diff
@@ -39,6 +39,11 @@ NS_ASSUME_NONNULL_BEGIN
 */
 @property (nonatomic, readonly) MSALDeviceMode deviceMode;
 
+/**
+ Additional device information
+*/
+- (NSDictionary *) extraDeviceInformation;
```

Comment:
nit: perhaps a readonly property would be more appropriate.

Reply:
sure, changed :)